### ISSUE TITLE
createPlayer requires uidResolver

### ISSUE THREAD

**Issue by spunkedy** [a-ignatov-parc/tvdml#8]
createPlayer requires uidResolver.

![screen shot 2017-01-02 at 10 05 32 pm](https://cloud.githubusercontent.com/assets/690735/21600228/6e85b824-d13e-11e6-9c8c-cf2bd4a90933.png)

The documentation says that the url is used by default. 

example not working

```
import * as TVDML from 'tvdml';

export default function HandlePlay(playObject){
  let urls = playObject.playList.slice(playObject.playItemIndex).map( singleItem => {
    return {
      url: singleItem.hdUri,
      title: singleItem.name
    }
    });
  console.log("Creating Player");
  console.log(urls);
  TVDML.createPlayer({
    items: urls
  })
  .then(player => player.play())
  .catch(err => console.log(err));
}


```

**Comment by spunkedy** [comment]
If I have time I will do a PR to fix it. Not sure when that will be

**Comment by a-ignatov-parc** [comment]
Ahh... that was a stupid mistake with [default arguments](https://github.com/a-ignatov-parc/tvdml/blob/master/src/render/player.js#L100). `uidResolver` by default is `null` and `null` is handled in ES6 as proper value.

Fixed in https://github.com/a-ignatov-parc/tvdml/commit/df1e2ecddd9d43194aa6517a71e8050d16a607bc and published fix to npm as `v1.0.2`.

Thanks for your help!

**Comment by a-ignatov-parc** [comment]
Found out that first fix was incorrect. Proper fix shipped in `v1.0.3`.

**Comment by spunkedy** [comment]
cool
